docs/reference/ingest/apis/put-pipeline.asciidoc

```diff
@@ -76,7 +76,53 @@ Processors run sequentially in the order specified.
 `version`::
 (Optional, integer)
 Version number used by external systems to track ingest pipelines.
+
 +
 This parameter is intended for external systems only. {es} does not use or
 validate pipeline version numbers.
+
+`_meta`::
+(Optional, object)
+Optional user metadata about the ingest pipeline. May have any contents. This
```

Comment:
Nit. I don't think `user` is needed here. If we want to keep it, I'd probably use `user-defined metadata`.
```suggestion
Optional metadata about the ingest pipeline. May have any contents. This
```